src/nationalcareers_toolkit/assets/src/frontend/js/compui/apps/dfc-app-findacourse.js

```diff
@@ -75,6 +81,12 @@ $(document).ready(function () {
                 if (data.payload === "true") {
                     $('.find-a-course-page #distance-block').show();
                     $("#orderBy-Input")[0].options.add(new Option("Distance", "Distance"));
+                    var a = document.getElementById('fac-result-list').getElementsByTagName('a'),
```

Comment:
is this trailing coma correct?

Reply:
Yes, this is vanilla js.

Reply:
The whole method is:

var a = document.getElementById('fac-result-list').getElementsByTagName('a'),
             length = a.length;